tutorials/promscale-run-queries.md

```diff
@@ -0,0 +1,279 @@
+          memory_used.time <= memory_allowed.end_time AND
+          eq(memory_used.labels,memory_allowed.labels)) 
+WHERE memory_used.value != 'NaN'   
+GROUP BY container 
+ORDER BY percent_used_p99 ASC 
+LIMIT 100;
+```
+
+A sample output for the above query is as follows:
+```sql
+ container                      |   percent_used_p99    |   total
+--------------------------------+-----------------------+---------------
+cluster-overprovisioner-system  | 6.961822509765625e-05 |  4294967296
+sealed-secrets-controller       |   0.00790748596191406 |  1073741824
+dumpster                        |    0.0135690307617187 |   268435456
+```
+While the above example requires the installation of `cAdvisor`, it's just an example of the sorts of sophisticated analysis enabled by Promscale's support to query your data in SQL.
+
+### 4.2 PromQL Queries in Promscale [](promql-querying-using-promscale)
+Promscale can also be used as a Prometheus data source for tools like [Grafana][grafana-homepage] and [PromLens][promlens-homepage]. 
+
+We'll deomstrate this by connecting Promscale as a Prometheus data source in [Grafana][grafana-homepage], a popular open-source visualization tool.
+
+First, let's install Grafana via [their official Docker image][grafana-docker]:
+
+```bash
+docker run -d \
+  -p 3000:3000 \
+  --network promscale-timescaledb \
+  --name=grafana \
+  -e "GF_INSTALL_PLUGINS=grafana-clock-panel,grafana-simple-json-datasource" \
+  grafana/grafana
+```
+
+Next, navigate to `localhost:3000` in your browser and enter `admin` for both the Grafana username and password, and set a new password. Then navigate to `Configuration > Data Sources > Add data source > Prometheus`.
+
+Finally, configure the data source settings, setting the `Name` as `Promscale` and setting the `URL` as`http://<PROMSCALE-IP-ADDR>:9201`, taking care to replace `<PROMSCALE-IP-ADDR>` with the IP address of your Promscale instance. All other settings can be kept as default unless desired.
+
+To find the Promscale IP address, run the command `docker inspect promscale` (where `promscale` is the name of our container) and find the IP address under `NetworkSettings > Networks > IPAddress`.
+
+Alternatively, we can set the `URL` as `http://promscale:9201`, where `promscale` is the name of our container. This method works as we've created all our containers in the same docker network (using the flag `-- network promscale-timescaledb` during our installs).
+
+After configuring the Promscale as a datasource in Grafana, all that's left is to  create a sample panel using `Promscale` as the datasource.  The query powering the panel will be written in PromQL. The sample query below shows the average rate of change in the past 5 minutes for the metric `go_memstats_alloc_bytes`, which measures the Go's memory allocation on the heap from the kernel:
+```bash
+rate(go_memstats_alloc_bytes{instance="localhost:9090"}[5m])
+```
+
+#### Sample output:
+
+![](https://i.imgur.com/vZPGsDu.png)
```

Comment:
This image breaks out of standard formatting b/c it doesn't have the `class` attached to the `<img>` tag.  Needs to be in standard format for showing images

![image](https://user-images.githubusercontent.com/12288273/107259712-564cf280-6a0b-11eb-8b74-dd91abb0b118.png)

Reply:
What is the standard image format -- is there a reference I can follow? Will look in other docs...